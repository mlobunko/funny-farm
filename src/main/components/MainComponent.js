import React from 'react';
import PropTypes from 'prop-types';

import Rotate from './Rotate';
import Content from './ContentContainer';

export const MainComponent = ({ isLandscape }) => (
  <div>{isLandscape ? <Content /> : <Rotate />}</div>
);

export default MainComponent;
MainComponent.displayName = 'Main';

MainComponent.propTypes = {
  isLandscape: PropTypes.bool
};
