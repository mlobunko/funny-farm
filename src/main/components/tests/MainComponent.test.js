import React from 'react';
import { shallow } from 'enzyme';

import { MainComponent } from '../MainComponent';

describe('render', () => {
  it('renders content', () => {
    const wrapper = shallow(<MainComponent isLandscape />);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders rotate', () => {
    const wrapper = shallow(<MainComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});
