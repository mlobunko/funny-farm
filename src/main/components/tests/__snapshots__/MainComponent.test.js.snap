// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders content 1`] = `
<div>
  <Connect(ContentContainer) />
</div>
`;

exports[`render renders rotate 1`] = `
<div>
  <Rotate />
</div>
`;
